types/QuantityMeasurementType.json

```diff
@@ -0,0 +1,21 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "/types/QuantityMeasurementType",
+    "description": "Quantity measures",
+
+    "type" : "object",
+   
+     "properties" : {
+        "value" : {
+            "type" : "number"
+        },
+        "unit" : {
+            "description" : "UNCEFACT unit",
+            "type" :"string"
```

Comment:
Should this be an enum?

Reply:
We'll need to decide what the set is, or simply specify its a code from the UN/CEFACT annex III codes. Depends where we use this.